### ISSUE TITLE
Package is not available on nuget?

### ISSUE THREAD

**Issue by mattvb1** [dotnet/AzCopyNet#16]
I tried searching for this package on nuget.org but can't find it. The nuget package links in the ReadMe reference a private nuget feed at https://dev.azure.com/xiaoyuz0315/BigMiao.

Is there a reason why this isn't on nuget.org or am I just missing something? 
We can't use a private nuget, which is unfortunate as the package looks great.

**Comment by LittleLittleCloud** [comment]
Yeah we didn't release it to nuget on behave of Microsoft because it's mostly a toy project and is mostly for internal use.

If you are looking for azure storage client, I recommend [Azure Blob Storage client](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-dotnet). AFAIK It's also powered by azcopy and is maintained by Microsoft.
